src/components/rotation.rs

```diff
@@ -101,6 +101,11 @@ impl Rotation {
         }
     }
 
+    /// Creates a [`Rotation`] from sine and cosine in radians
```

Comment:
Just to be slightly more descriptive and consistent:
```suggestion
    /// Creates a [`Rotation`] from the sine and cosine of an angle in radians.
```